.github/workflows/android.yml

```diff
@@ -67,56 +67,3 @@ jobs:
         # Copyp AAR to S3
         cp executorch.aar artifacts-to-be-uploaded/tiktoken_$EXECUTORCH_USE_TIKTOKEN/
         cp executorch-llama.aar artifacts-to-be-uploaded/tiktoken_$EXECUTORCH_USE_TIKTOKEN/
-
-  # Upload the app and its test suite to S3 so that they can be downloaded by the test job
-  upload-artifacts:
```

Comment:
Just FYI, an `if: false` would also skip the job if you still want to keep it around but not run it

Reply:
Yeah, I would prefer running it only on emulator with tiny model after validating the build. Everything on the critical path should be stable and efficient, so will move the benchmarking workflow to a non-critical path anyway.